simpeg_drivers-assets/uijson/magnetic_vector_inversion.ui.json

```diff
@@ -858,6 +858,16 @@
         "value": 1,
         "visible": false
     },
+    "solver_type": {
```

Comment:
Doesn't belong here. None of the PF use a direct solver. You could set it to visible False if that simplifies things.

Reply:
flattened to key/value.